### ISSUE TITLE
TypeError: Can't instantiate abstract class CustomTensorFlowModelWrapper with abstract methods get_grad

### ISSUE THREAD

**Issue by aaur0** [QData/TextAttack#365]
It seems the [change](https://github.com/QData/TextAttack/blob/c04f8f4b2a4396f70daaf9ebcbb9924fbb015cbf/textattack/models/wrappers/model_wrapper.py#L19) introduced in  v0.2.12 expects the **_get_grad_** method to be implemented for creating the CustomTensorFlowModelWrapper.  This has broken the [Example_0_tensorflow.ipynb notebook](https://github.com/QData/TextAttack/blob/3390a18c52402f75f0ea1a3ea5cf9e9fb90006c1/docs/2notebook/Example_0_tensorflow.ipynb). 

`CustomTensorFlowModelWrapper(model)(['I hate you so much', 'I love you'])`

returns the following error message

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-71-0d55636c5b88> in <module>
----> 1 model_wrapper = CustomTensorFlowModelWrapper(model)
      2 CustomTensorFlowModelWrapper(model)(['I hate you so much', 'I love you'])

TypeError: Can't instantiate abstract class CustomTensorFlowModelWrapper with abstract methods get_grad
```

**Comment by Opdoop** [comment]
I run into the same Error in https://github.com/QData/TextAttack/issues/354#issuecomment-735839449. Maybe there is some other incompatible with the class `CustomTensorFlowModelWrapper` or `ModelWrapper`. Hope you could solve this bug.

**Comment by aaur0** [comment]
For now, I have downgraded to 0.2.10 and it seems to have fixed the issue.

**Comment by jinyongyoo** [comment]
@aaur0 Hi this is because `get_grad` was set as an abstract method, which means that any class extending `ModelWrapper` needs to define it. But since `get_grad` is only needed only for operations that require gradient access, seems like this was an oversight by us. Thanks for reporting this. We'll get it fixed by next release.

**Comment by jinyongyoo** [comment]
Fixed by #381
